### PR TITLE
Remove problematic protocol rewrite from YouVersion.xml

### PR DESCRIPTION
**Pull request by cschanaj**
Part of #12297

### CODE REVIEW

File: src/chrome/content/rules/YouVersion.xml [hunk 1]
```diff
@@ -1,29 +0,0 @@
...
-	<target host="*.biblesociety.co.za" />
-	<target host="youversion.com" />
-	<target host="*.youversion.com" />
-
-
-	<securecookie host="^(?:www)?\.(?:bible|youversion)\.com$" name=".+" />
-	<securecookie host="^\.biblesociety\.co\.za$" name="^_youversion-web_session$" />
-
-
-	<rule from="^http?://(?:www\.)?(bible|yourversion)\.com/"
```
Comment (cschanaj) [inline src/chrome/content/rules/YouVersion.xml]: :bug: :bug: :bug:

Comment (cschanaj) [reply src/chrome/content/rules/YouVersion.xml]: cc @Hainish @J0WI @jeremyn @Bisaloo

Comment (cschanaj) [reply src/chrome/content/rules/YouVersion.xml]: Any chance for merging before #16122?

File: src/chrome/content/rules/Bible.com.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+		- share.bible.com
+
+		SSL peer certificate was not OK:
+		- free.bible.com
+-->
+<ruleset name="Bible.com">
+	<target host="bible.com" />
+	<target host="www.bible.com" />
+	<target host="app.bible.com" />
+	<target host="events.bible.com" />
```
Comment (Bisaloo) [inline src/chrome/content/rules/Bible.com.xml]: nodejs.bible.com with test url http://nodejs.bible.com/assets/style.main.4e4c1183999a10efd085c69bc1cf27d0.css

File: src/chrome/content/rules/BibleSociety.co.za.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<ruleset name="BibleSociety.co.za">
```
Comment (Bisaloo) [inline src/chrome/content/rules/BibleSociety.co.za.xml]: I get MCB warnings

**Comment by Bisaloo** [comment]
Otherwise LGTM

**Comment by cschanaj** [comment]
@Bisaloo Travis is passing now. thanks!

**Pull request merged**